config/environments/staging.rb

```diff
@@ -1,13 +1,6 @@
 require_relative "production"
 
 Mail.register_interceptor(
-  RecipientInterceptor.new(ENV.fetch("EMAIL_RECIPIENTS"))
```

Comment:
Why did you remove the `ENV.fetch`? Just curious.

Reply:
I just copied the full block from another project.
In addition, it wasn't either specifying a default value for the fetch nor requiring an exception to be thrown, so it doesn't really matter.